listener.go

```diff
@@ -165,7 +164,7 @@ func (l *listener) WrapExpiration(days int) net.Listener {
 	go func() {
 		// busy-wait for certificate preload to complete
```

Comment:
Nit: correct the comment as well
```suggestion
		// loop on short sleeps until certificate preload completes
```

Reply:
updated, thanks.